react-ui/src/pages/contest/ContestUnderReview.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * ??? When is this displayed?
+ * ??? onClick={mutate} vs onClick={() => mutate()}
```

Comment:
They're functionally the same, but the first one is cleaner. Perhaps there was an additional line that got removed, which is why I had an arrow function there before.